docs/lib/getting-started/fragments/android/add-api.md

```diff
@@ -1,42 +1,205 @@
-                "amplifyDatasource": {
-                    "endpointType": "GraphQL",
-                    "endpoint": "https://<YOUR-GRAPHQL-ENDPOINT>.appsync-api.us-west-2.amazonaws.com/graphql",
-                    "region": "us-west-2",
-                    "authorizationType": "API_KEY",
-                    "apiKey": "<YOUR API KEY>"
-                }
+Now that your have DataStore persisting data locally, in the next step you'll connect it to the cloud. With a couple of commands, you'll create an AWS AppSync API and configure DataStore to synchronize its data to it.
+
+1. Edit your `onCreate` method to remove all of the code you added. Your `onCreate()` not include any Amplify code.
+
+1. Add the following at the bottom of the `onCreate` method: 
+
+  <amplify-block-switcher>
+  <amplify-block name="Java">
+
+  ```java
+    try {
+      Amplify.addPlugin(new AWSDataStorePlugin());
+      Amplify.addPlugin(new AWSApiPlugin());
+      Amplify.configure(getApplicationContext());
+
+      Log.i("Tutorial", "Initialized Amplify");
+  } catch (AmplifyException e) {
+      Log.e("Tutorial", "Could not initialize Amplify", e);
+  }
+
+  Todo item = Todo.builder()
+          .name("Tidy up the office")
+          .description("Organize books, vacuum, take out the trash")
+          .build();
+
+    Amplify.DataStore.save(
+            item,
+            success -> Log.i("Tutorial", "Saved item: " + success.item.getName()),
+            error -> Log.e("Tutorial", "Could not save item to DataStore", error)
+    );
+  ```
+
+  </amplify-block>
+
+  <amplify-block name="Kotlin">
+
+  ```kotlin
+    try {
+      Amplify.addPlugin(new AWSDataStorePlugin())
+      Amplify.addPlugin(new AWSApiPlugin())
+      Amplify.configure(applicationContext)
+
+      Log.i("Tutorial", "Initialized Amplify")
+  } catch (e: AmplifyException) {
+      Log.e("Tutorial", "Could not initialize Amplify", e)
+  }
+
+  val item: Todo = Todo.builder()
+        .name("Tidy up the office")
+        .description("Organize books, vacuum, take out the trash")
+        .build()
+
+   Amplify.DataStore.save(
+          item,
+          { success -> Log.i("Tutorial", "Saved item: " + success.item.name) },
+          { error -> Log.e("Tutorial", "Could not save item to DataStore", error) }
+  )
+  ```
+
+  </amplify-block>
+  </amplify-block-switcher>
+
+1. Open up a terminal window. You can use an external terminal or the integrated terminal in Android Studio.
+
+1. Run `amplify configure`. This step will configure a new AWS user in your account for Amplify to manage resources on your behalf.
+
+    ```bash
+    amplify configure
+    ```
+
+1. First, it will open up the AWS Management Console in your web browser and prompt you to login.
+
+    ```console
+    Follow these steps to set up access to your AWS account:
+
+    Sign in to your AWS administrator account:
+    https://console.aws.amazon.com/
+    Press Enter to continue
+    ```
+
+    Sign in and press **Enter**.
```

Comment:
Might be worth calling out you sign in on the browser and then return to the terminal to press enter similar to what you did further down